gpm_core/src/store_project.rs

```diff
@@ -0,0 +1,111 @@
+use std::fs::File;
+use std::io;
+use std::io::Read;
+use std::path::{Path, PathBuf};
+
+use crate::package::{Package, PackageInformation, PackageInformationExtraData};
+use crate::TOML_CONFIG_PATH;
+
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize)]
+struct StoredPackageInformation {
+    creator: String,
+    identifier: String,
+    version: String,
+    display_name: String,
+    description: String,
+    license: String,
+
+    #[serde(default)]
+    website_url: Option<String>,
+    #[serde(default)]
+    dependencies: Vec<String>,
```

Comment:
same as above, about the dependencies with IDs or with names. The specs says IDs and here it's a string.